lib/ui.ex

```diff
@@ -30,13 +29,13 @@ defmodule Ui do
     board
     |> Board.rows
     |> Enum.map(&draw_line/1)
-    |> Enum.join(line)
+    |> Enum.join(line(Board.dimension(board)))
```

Comment:
I would extract creating the line into a temp variable. Otherwise it reads a little weird...